packages/integrations/vercel/README.md

```diff
@@ -262,6 +262,82 @@ You can use Vercel middleware to intercept a request and redirect before sending
 > **Warning**
 > **Trying to rewrite?** Currently rewriting a request with middleware only works for static files.
 
+### Vercel Edge Middleware with Astro middleware
+
+The `@astrojs/vercel/serverless` adapter can automatically create the vercel edge middleware when you have an Astro middleware in you code base.
```

Comment:
```suggestion
The `@astrojs/vercel/serverless` adapter can automatically create the Vercel Edge middleware from an Astro middleware in your code base.
```
Would it be correct to say it like this? That you write it in Astro, but it gets "turned into" Vercel middleware code, executed by them? If so, I think this explains that.

Reply:
I believe so!